src/TreeBuilderManager.cpp

```diff
@@ -99,10 +99,12 @@ std::string TreeBuilderManager::doJob() {
             this->alphType =
                 (TreeBuilderManager::AlphabetType)seqReader->getAlphType();
             fprintf(stderr, "Calculating distances....\n");
-            DistanceCalculator *distCalc = new DistanceCalculator(
-                seqs, (DistanceCalculator::AlphabetType)alphType,
-                (DistanceCalculator::CorrectionType)corrType,
-                seqReader->numSeqs, this->newDistanceMethod);
+            void *distCalcMem = _mm_malloc(sizeof(DistanceCalculator), 32);
+            DistanceCalculator *distCalc = static_cast<DistanceCalculator*>(distCalcMem);
+            new(distCalc) DistanceCalculator(
+                    seqs, (DistanceCalculator::AlphabetType)alphType,
+                    (DistanceCalculator::CorrectionType)corrType,
+                    seqReader->numSeqs, this->newDistanceMethod);
```

Comment:
Do we still need to do this?

Reply:
I think we do, until/if we change all the field allocations to use aligned_alloc.